### ISSUE TITLE
Custom Package Issue

### ISSUE THREAD

**Issue by milezzz** [TelescopeJS/Zeiss#4]
Playing with the latest `Devel` branch of Telescope and latest Zeiss I seem to have a problem with the custom packages.  I enable the `my-custom-package` okay but for some reason the .scss and other files seem to load and function as expected, but not the template replacements prefaced with `custom_`...

From my packages.js inside `my-custom-package`: 

`'lib/client/templates/custom_post_title.html',`
`'lib/client/templates/custom_post_title.js',`

They're enabled okay but the changes do not show in my client.  I must be missing something.

**Comment by SachaG** [comment]
What if you load your package after Zeiss (make it depends on the Zeiss package)? Does that change anything?

**Comment by milezzz** [comment]
Actually tried both ways but seems to make no difference.  Right now it is:

`############ Your Custom Packages ############`

`telescope:zeiss`
`tu-mods`

**Comment by SachaG** [comment]
What if you defined your own custom prefix?

**Comment by milezzz** [comment]
> > What if you defined your own custom prefix?

I managed to do this - I used "custom_", but I still can't alter any Zeiss templates.  Or rather seems the Zeiss ones load after mine.  Hmm I must be overlooking something.  

Hmm just noticed all the videos you guys have posted to youtube.  Nice.  Going to roll through those before going any further ;)

**Comment by SachaG** [comment]
What does your `Telescope.config.customPrefixes` array look like?

**Comment by milezzz** [comment]
`Array [ "custom_", "zss_" ]`

Both in there. 

The videos are very well done!  Still going through them...

I'm still missing a very key piece of information though.  Where are the zeiss template, css, js, etc. files located locally?  I was hoping they would be in `/packages` somewhere but after quickly searching the only zeiss files I can see are buried deep in `.meteor`:

`./.meteor/local/build/programs/server/packages/telescope_zeiss.js.map`
`./.meteor/local/build/programs/server/packages/telescope_zeiss.js`
`./.meteor/local/build/programs/web.browser/packages/telescope_zeiss.js.map`
`./.meteor/local/build/programs/web.browser/packages/telescope_zeiss.js`

**Comment by SachaG** [comment]
Although in theory `custom_` should still work, I haven't really tested it in depth yet. So maybe the easiest solution is just for you to add your own new custom prefix that will take priority over both?

And yes, that's where the files are stored, along with the other Meteor packages. You shouldn't really have to worry about that though.

**Comment by milezzz** [comment]
> > So maybe the easiest solution is just for you to add your own new custom prefix that will take priority over both?

Yep tried that but for some reason the zeiss theme templates appear to be getting loaded last.  Also tried these two:

`'lib/client/templates/post_title.html',`
`'lib/client/templates/post_title.js',`

I thought for sure they would load...but same result.  I even rebuilt my VM to make sure it's not a botched install.  

I'm sort of out of ideas...I'll have another look when I get some free time but maybe someone else can chime in if custom packages are working okay with the latest builds?

**Comment by SachaG** [comment]
Can you upload your repo somewhere on GitHub so I can take a look directly?

**Comment by asennoussi** [comment]
Having the same issue. 
Found a solution ?
